helm-charts/README.md

```diff
@@ -151,6 +151,7 @@ A Helm chart for APK components
 | wso2.apk.dp.gatewayRuntime.deployment.router.strategy | string | `"RollingUpdate"` | Deployment strategy |
 | wso2.apk.dp.gatewayRuntime.deployment.router.imagePullPolicy | string | `"Always"` | Image pull policy |
 | wso2.apk.dp.gatewayRuntime.deployment.router.image | string | `"wso2/router:1.0.0"` | Image |
+| wso2.apk.dp.gatewayRuntime.deployment.router.adminInterfaceEnabled | bool | `true` | Envoy proxy admin interface enabled or not |
```

Comment:
We dont need this config. Added a comment in here - https://github.com/wso2/docs-apk/pull/382#discussion_r1342326373